### DISCUSSION TITLE
Written a react-fabric

### DISCUSSION THREAD

**Discussion by vaynevayne** [fabricjs/fabric.js#10325] [Ideas]
https://github.com/cs-open/react-fabric
I have developed a react-fabric library for fabric. Can I add it to the official recommendation list, or add it to the official project group to develop it together? @asturur

**Comment by usamshen** [comment]
Thanks for sharing! This is interesting project, did you test the
performance, for example five  thousands of various of objects on one
canvas?

**Reply by vaynevayne** [reply]
I haven't actually tested it. When mounted, canvas.renderAll will only be executed once. Performance depends on react and fabricjs itself.
When updating, react only rerenders modified components
Welcome to add tests @usamshen

**Comment by zhe-he** [comment]
发布fabric-react不需要经过社区的同意，直接发布即可，具体方式可以问deepseek，发布前请关闭全局淘宝镜像。
Publishing fabric-react does not require community approval and can be released directly. For specific methods, you can ask DeepSeek. Before publishing, please disable the global Taobao mirror.

**Reply by vaynevayne** [reply]
主要是同名了 后来 我起了个组织名
